v1/plugin/stream_collector_proxy.go

```diff
@@ -199,9 +223,14 @@ func (p *StreamProxy) setMaxMetricsBuffer(i int64) {
 	p.maxMetricsBuffer = i
 }
 
-func sendReply(metrics []*rpc.Metric, stream rpc.StreamCollector_StreamMetricsServer) {
+func sendReply(taskID string, metrics []*rpc.Metric, stream rpc.StreamCollector_StreamMetricsServer) {
 	if len(metrics) == 0 {
-		fmt.Println("No metrics available to send")
+		log.WithFields(
+			log.Fields{
+				"_block": "sendReply",
+				"taskid": taskID,
```

Comment:
The field for task is named `"task-id"` (task-dash-id) in all log statements in snap, so please rename all fields to have consistent logs.